jobs/geo/core.py

```diff
@@ -1,44 +1,90 @@
 from __future__ import annotations
```

Comment:
I recommend renaming this file to `tiger.py`.

```bash
git mv core.py tiger.py
```
to retain history

Reply:
done